### PR TITLE
Api provides the ability to get a specific article

### PR DESCRIPTION
**Pull request by felixoppongwillborg**
### User Story
```
As a newsroom api
in order for user to request specific article
i would like to be able to provide an endpoint for showing an article.
```
## Changes proposed in this pull request:
* adding show action, adding test for show request. added article content to DB and added show action to routes.
*added a rescue in show action 

## What I have learned working on this feature:
* practice adding show action and adding new table to DB.
*adding rescue and modifying response.

### CODE REVIEW

File: app/controllers/api/v1/articles_controller.rb [hunk 1]
```diff
@@ -3,4 +3,8 @@ def index
     articles = Article.all
     render json: articles, each_serializer: ArticlesIndexSerializer
   end
+  def show
+    article = Article.find(params[:id])
```
Comment (oliverochman) [inline app/controllers/api/v1/articles_controller.rb:7]: what happens if the api cant find the article?

File: app/serializers/articles_index_serializer.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 class ArticlesIndexSerializer < ActiveModel::Serializer
-  attributes :id, :title, :lead
+  attributes :id, :title, :lead, :content
 end
```
Comment (emtalen) [inline app/serializers/articles_index_serializer.rb:3]: Sooo... now you are sending the content with the index action? You should probably generate a new serializer for the show action.

File: spec/requests/api/v1/articles/show_article_spec.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+RSpec.describe "GET /v1/articles", type: :request do
+  let!(:article) { create(:article, title: 'The first article', lead: 'This is the first article lead', content: 'This is the first article content') }
+  let!(:article2) { create(:article) }
```
Comment (oliverochman) [inline spec/requests/api/v1/articles/show_article_spec.rb:3]: why do you need this article?

Comment (emtalen) [reply spec/requests/api/v1/articles/show_article_spec.rb:3]: We were talking about how the response looks like and I told them to leave it to test that they get back the article that they are requesting

**Pull request merged**